client/src/modules/journal/journal.js

```diff
@@ -321,6 +331,7 @@ function JournalController(Journal, Sorting, Grouping,
         vm.gridOptions.data = transactions.preprocessJournalData(records);
 
         transactions.applyEdits();
+        vm.gridApi.core.notifyDataChange(uiGridConstants.dataChange.ALL);
```

Comment:
This looks like a good solution if it works! I wonder why the grid doesn't register that data has changed in this case? 

This is concerning as either this should happen automatically or it should be the responsibility of the service that has changed the data to make this notify. 

Could you leave an `@TODO` comment so that this can be investigated in the future? 
```//@TODO investigate why footer totals aren't updated automatically on data change```

This solution will work well for now.